cluster-logging.md

```diff
@@ -104,11 +105,25 @@ The "Storage" column shows whether the attribute is stored into a LokiStack usin
 | `service.name` | resource | journal | stream label | |
 | `systemd.t.*` | log | journal | structured metadata | |
 | `systemd.u.*` | log | journal | structured metadata | |
+| `k8s.ovn.component` | resource | OVN logs | stream label | The OVN component generating the log|
+| `k8s.ovn.sequence` | log | OVN logs | structured metadata | The sequence number indicating the log entry's order|
+| `k8s.ovn.message` | log | OVN logs | structured metadata | The main content of the log entry, can containing key-value pairs|
```

Comment:
@alanconway either "body" is equal to this or the entire original message meaning we don't need "message".  The 'message.*' is parsing the value

Reply:
+1, no need for "message" body is the message.

Reply:
done